index.html

```diff
@@ -0,0 +1,225 @@
+        </a>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/servicesicon.svg" alt="food and services icon">
+          <h3><a href="#" class="program-cards-title orange">Food & Services</a></h3>
+          <p>You will have a variety of services available like food and souvenir stands. See inside
+            for more details</p>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/exibitionicon.svg" alt="Exhibition icon">
+          <h3><a href="#" class="program-cards-title orange">Inti Raymi Exhibition</a></h3>
+          <p>Join the exhibition to learn more about this incredible celebration. It is for all public that attend the
+            festival!</p>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/programicon.svg" alt="Forum icon">
+          <h3><a href="#" class="program-cards-title orange">Forum</a></h3>
+          <p>Foreigners and nationals are welcome in this festival! Get in touch with other people interested in this event.</p>
+        </div>
+
+        <button class="join-inti">Join Inti Raymi Celebration 2023</button>
+      </div>
+
+      <div class="desktop-see d-none d-md-flex justify-content-center py-5 mt-5">
+        <h4 href="#" class="desktop-see-program">SEE THE WHOLE PROGRAM</h4>
+      </div>
+
+    </section>
+
+    <div class="perform-cont">
+
+    </div>
+
+    <section class="main-performers" id="performers-list ">
+
+      <div class="d-flex justify-content-center" id="performer-title">
+        <h2 class="perf-title">Performers</h2>
+        </div>
+
+      </div>
+
+      <div class="container-performers p-0 d-flex ">
+        
+        <article id="all-performers" class="d-flex gap-4">
+
+        </article>
+      </div>
+    </section>
+
+    <button type=button class="more-less flex" id="showMore">MORE</button>
+
+    </section>
+    <section class="sponsor-cont">
+      <div class="titlePartners flex">
+        <h2 class="">Partners</h2>
+      </div>
+      <div id="sponsors">
+        <i class="sponsor"><img class="w1" src="./capstoneimgs/airbnb.svg" alt="Airbnb logo."></i>
+        <i class="sponsor"><img class="w2" src="./capstoneimgs/produbancogrey-removebg-preview.png"
+            alt="produbanco logo."></i>
+        <i class="sponsor"><img class="w2" src="./capstoneimgs/bancopichincha-removebg-preview.png"
+            alt="Banco Pichincha logo"></i>
+        <i class="sponsor"><img class="w3" src="./capstoneimgs/metropolitant-removebg-preview.png"
+            alt="Metropolitan touring logo"></i>
+        <i class="sponsor"><img class="w2" src="./capstoneimgs/confiamed-removebg-preview.png" alt="confiamed logo"></i>
+      </div>
+    </section>
+
+    <footer class="desktop-footer white">
+      <img class="footer-icon m-2 px-5" src="./capstoneimgs/logo.png" alt="Logo">
+      <p class="footer-p m-2 px-4">Inti Raymi Music Festival 2023</p>
+    </footer>
```

Comment:
- [ ] Kindly consider adding a top border to your footer to make it look better and also consider aligning your text with the logo properly.
![zx](https://user-images.githubusercontent.com/96683843/226715078-bf66aa8d-7021-4c40-98ae-697cae6604fd.jpg)

Reply:
Thanks, done!